shell.c

```diff
@@ -150,4 +184,9 @@ run_cpu(i8080 *cpu, int cycles)
         }
     }
   return cycles;
+
+  // Destroy window
+  SDL_DestroyWindow(window);
```

Comment:
I'm still confused why this is here. We want sdl to destroy the window after every time run_cpu is called? This function isn't even where any of the graphics are drawn.

Reply:
Seems like it belongs at the bottom of main, or perhaps in the loop there running conditional on whichever opcode handles quitting the game...

Reply:
Can we move it outside the loop but inside main? Not inside run_cpu?